rust/migration/src/m20220101_000001_create_table.rs

```diff
@@ -17,25 +29,227 @@ impl MigrationTrait for Migration {
                             .auto_increment()
                             .primary_key(),
                     )
+                    .col(ColumnDef::new(Job::Hash).ezblob().unique_key())
                     .col(ColumnDef::new(Job::Cmd).string().not_null())
-                    .col(ColumnDef::new(Job::Env).string().not_null())
+                    .col(ColumnDef::new(Job::Env).ezblob())
+                    .col(ColumnDef::new(Job::Cwd).string().not_null())
+                    .col(ColumnDef::new(Job::Stdin).string().not_null())
+                    .col(ColumnDef::new(Job::IsAtty).boolean().not_null())
+                    .col(ColumnDef::new(Job::HiddenInfo).ezblob())
+                    .col(ColumnDef::new(Job::Stdout).ezblob())
+                    .col(ColumnDef::new(Job::Stderr).ezblob())
+                    .col(ColumnDef::new(Job::Status).integer().not_null())
+                    .col(ColumnDef::new(Job::Runtime).double().not_null())
+                    .col(ColumnDef::new(Job::Cputime).double().not_null())
+                    .col(ColumnDef::new(Job::Memory).big_unsigned().not_null())
+                    .col(ColumnDef::new(Job::IBytes).big_unsigned().not_null())
+                    .col(ColumnDef::new(Job::OBytes).big_unsigned().not_null())
```

Comment:
Is this missing "Runner hash"?

Reply:
hidden info is a more generic name for `runner_hash` that we can use to cover more things.